vagrant/nodes.statedump.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: all
+  become: yes
+  become_method: sudo
+  vars_files:
+    - ansible/config.yml
+  roles:
+    - logs.collect
+
+- hosts: cluster
```

Comment:
Isn't there anything in the client side that could be interesting ?

Right now we only use sit-test-cases. I'm not sure if it's the case, but there could be some client side configuration for the tests.

Reply:
I have included a test configuration data file for now.